src/OnlineRetailPortal.Mock/Sort.cs

```diff
@@ -0,0 +1,28 @@
+﻿using OnlineRetailPortal.Contracts;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace OnlineRetailPortal.Mock
+{
+    public static class Sort
+    {
+        public static List<Product> ApplySort(this List<Product> productList , Contracts.Sort sort)
+        {
+            if (sort == null)
+                return productList;
+            return sort.Order == "Asc" ? productList.SortAsc(sort.Type) : productList.SortDesc(sort.Type);
+
+        }
+        public static List<Product> SortAsc(this List<Product> products, string sortBy)
```

Comment:
club both the methods as single sort function which will sort the products based on inputs.
SortAsc and SortDesc as public methods are not really needed.
You can have one method which sorts the data and then delegates the sorting to these private methods